nc-games-fe/src/Components/Vote-On-Review.jsx

```diff
@@ -0,0 +1,29 @@
+import React, {useState, useEffect} from 'react';
+import { updateReviewVotes } from './utils';
+
+const VoteButtons = ({ reviewId, setVotes }) => {
```

Comment:
Just on your component name and file name - I'd have these the same. Also it's a little nit picky but this is a `VoteButton` (singular) not a `VoteButtons` (Plural), I'd also name the file the same so `VoteButton.jsx`